test/unit_tests/apps/teachers-digital-platform/js/survey/results-page-spec.js

```diff
@@ -1,55 +1,73 @@
 import { jest } from '@jest/globals';
-import Cookies from 'js-cookie';
-import surveys from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/tdp-surveys.js';
-import { resultsPage } from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/survey/result-page.js';
 import {
   ANSWERS_SESS_KEY,
   SURVEY_COOKIE,
 } from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/survey/config.js';
-import * as modals from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/modals.js';
-import * as initials from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/survey/initials.js';
-import * as obfuscation from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/obfuscation.js';
 import HTML_SNIPPET from '../../html/results-page.js';
 
-// eslint-disable-next-line import/no-unresolved
-import clipboardCopy from 'copy-to-clipboard';
-
 const $ = document.querySelector.bind(document);
 
-xdescribe('The TDP survey results page', () => {
+describe('The TDP survey results page', () => {
+  let initials, surveys, resultsPage, obfuscation;
+  const remove = jest.fn();
+  const init = jest.fn();
+  const close = jest.fn();
+  const copy = jest.fn();
```

Comment:
Same as the `remove` comment, can we prefix these with what module they are in reference to, e.g.:

```
cookieRemove
modalsInit
modalsClose
clipboardCopy
```

Reply:
Done in 0522503